mkt/extensions/migrations/0015_extension_author.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('extensions', '0014_soft_delete'),
```

Comment:
Is this a real dependency? Seems to me that it depends on nothing but `0001_initial`